.github/workflows/master.yml

```diff
@@ -177,6 +178,12 @@ jobs:
         mkdir -p ~/.m2
         ./dev/run-tests --parallelism 2 --modules "$MODULES_TO_TEST" --included-tags "$INCLUDED_TAGS" --excluded-tags "$EXCLUDED_TAGS"
         rm -rf ~/.m2/repository/org/apache/spark
+    - name: "Publish test report: ${{ matrix.modules }}"
+      if: always()
+      uses: scacap/action-surefire-report@v1
```

Comment:
I saw your comment in https://github.com/apache/spark/pull/29169#issuecomment-667876128
Quick question here: Is there another working plugin?

Reply:
Nope.. this one was the only one given my few tries. I'm trying to figure out the github token issue. I'll bring up more info later.